core/modules/system/system.module

```diff
@@ -1448,6 +1448,20 @@ function system_library_info() {
     ),
   );
 
+  // details-polyfill.
+  $libraries['details-polyfill'] = array(
+    'title' => 'Polyfill for the HTML5 `<details>` element',
+    'website' => 'https://github.com/rstacruz/details-polyfill',
+    'version' => '1.2.0',
+    'js' => array(
+      'core/misc/html5-details.js' => array(
+        'group' => JS_LIBRARY,
+        'weight' => -22,
+        'browsers' => array('IE' => 'lte IE 8', '!IE' => FALSE),
```

Comment:
Do we even need this? I think we dropped support for IE.

Reply:
Removed with [`fb39c4e` (#3629)](https://github.com/backdrop/backdrop/pull/3629/commits/fb39c4ee918d30de6082dc10d57eeb1aee917842)